src/widgets/company/ChosenUs/styled.ts

```diff
@@ -2,17 +2,22 @@ import { styled } from 'styled-components';
 
 import { StContainer } from '@/shared/styles/global';
 
+export const StSection = styled.section`
+  margin-bottom: 80px;
+  background: rgba(255, 255, 255, 0.7);
+`;
+
 export const StContainerInner = styled(StContainer)`
   display: flex;
   text-align: center;
   flex-direction: column;
-  padding: 46px 0 41px;
-  gap: 65px;
+  padding: 40px 0 54px;
+  gap: 26px;
 `;
 
 export const StList = styled.ul`
   display: grid;
   grid-template-columns: repeat(3, 346px);
   grid-template-rows: repeat(2, 280px);
```

Comment:
Замени здесь, пожалуйста. Наставник нам так посоветовал
```suggestion
  grid-template-columns: repeat(auto-fill, minmax(346px, 1fr));
  grid-auto-rows: 280px;
```